anonymizer.go

```diff
@@ -0,0 +1,40 @@
+package goanonymizer
+
+import (
+	"errors"
+	"net"
+)
+
+//Anonymize struct that hold mask for V4 and V6.
+type Anonymize struct {
+	v4Mask net.IPMask
+	v6Mask net.IPMask
+}
+
+//NewAnonymize returns instance of Anonymize.
+//If v4Mask and v6Mask are not provided, default values are used.
+func NewAnonymize(v4Mask, v6Mask string) *Anonymize {
```

Comment:
Please describe in the comment:
- what the function of v4Mask and v6Mask is
- what the default values are

Reply:
To instantiate a default anonymizer, too empty strings needs to be passed to `New`. That is not soo elegant.

To make it easier to instantiate an IPAnonymized with the defaults, we could either:
- add a new constructor with no arguments
- define a package var like  `var DefIPAnonymizer = New(DefaultIPv4Mask, DefaultIPv6Mask)`

Reply:
@fho, what about the "options" approach?
```go
defAn := goanonimizer.NewAnonimizer()
noAn := goanonimizer.NewAnonimizer(
    goanonimizer.WithIPV4Mask(9), // mask out the last 9 bits of IPv4 addresses
)
```

Reply:
I think it's a lot of overhead here. 
I don't see that we will have more options in the future.
Passing both parameters as args is very simple.

Reply:
btw currently v4Mask and v6Mask seem to only support to be a full network mask, only passing the number of bits that describe the network should not work.
We could change it to support both notations